docs/tutorials/01_introductory/viz_solar_system.py

```diff
@@ -239,10 +239,43 @@ def update_planet_position(r_planet, planet_actor, planet_track, cnt):
     return pos_planet
 
 
-def update_track(positions_planet, planet_track, planet_orbit_actor, cnt):
-    positions_planet[:cnt+1] = np.array(planet_track)
+def update_track(positions_planet, planet_track, planet_orbit_actor):
+    positions_planet[:] = np.array(planet_track)
     utils.update_actor(planet_orbit_actor)
 
+def calculate_path(r_planet, planet_track, cnt):
+    pos_planet = get_orbital_position(r_planet, cnt)
+    planet_track.append([pos_planet[0], 0, pos_planet[1]])
+
+##############################################################################
+# Calculating and updating the path/orbit before animation starts
+
+for i in range(2200):
+    calculate_path(r_mercury, mercury_track, i)
+    calculate_path(r_venus, venus_track, i)
+    calculate_path(r_earth, earth_track, i)
+    calculate_path(r_mars, mars_track, i)
+    calculate_path(r_jupiter, jupiter_track, i)
+    calculate_path(r_saturn, saturn_track, i)
+    calculate_path(r_uranus, uranus_track, i)
+    calculate_path(r_neptune, neptune_track, i)
+
+update_track(positions_mercury, mercury_track, mercury_orbit_actor)
+
+update_track(positions_venus, venus_track, venus_orbit_actor)
+
+update_track(positions_earth, earth_track, earth_orbit_actor)
+
+update_track(positions_mars, mars_track, mars_orbit_actor)
+
+update_track(positions_jupiter, jupiter_track, jupiter_orbit_actor)
+
+update_track(positions_saturn, saturn_track, saturn_orbit_actor)
+
+update_track(positions_uranus, uranus_track, uranus_orbit_actor)
+
+update_track(positions_neptune, neptune_track, neptune_orbit_actor)
+
```

Comment:
this block can be reduced with 3-4 lines of code which avoids duplication.

- use `zip`
- call `calculate_path` and `update_track` only once in your loop
- pass 2200 in your `calculate_path`

Reply:
If i use line actor then there is no need of `update_track`, 

And am I suppose to use `zip` for all the tracks list?